packages/flutter/lib/src/widgets/widget_inspector.dart

```diff
@@ -1435,92 +1424,69 @@ mixin WidgetInspectorService {
     return _followDiagnosticableChain(chain.reversed.toList());
   }
 
+  DiagnosticsSerialisationDelegate _configToDelegate(_SerializeConfig config) {
+    final Set<DiagnosticsNode> nodesCreatedByLocalProject = <DiagnosticsNode>{};
+    return DiagnosticsSerialisationDelegate(
```

Comment:
having the separate config and DiagnosticsSerializationDelegate classes seems like overkill. Wouldn't it be simpler to remove _SerializeConfig and just have DiagnosticssSerialiationDelegate implementation that implements this functionality?

Reply:
Yes, I'll change it to remove the _SerializeConfig.